sections/cart-notification-product.liquid

```diff
@@ -16,7 +16,7 @@
         {%- if settings.show_vendor -%}
           <p class="caption-with-letter-spacing light">{{ item.product.vendor }}</p>
         {%- endif -%}
-        <h3 class="cart-notification-product__name h4">{{ item.product.title | escape }}</h3>
+        <h3 class="cart-notification-product__name h4">{{ item.title | escape }}</h3>
```

Comment:
> We're already mentioning the variant/options picked in a nicer visual way and this is repeating that content.
Thanks for feedback! Yeah, agree that this might be not the nicest representation. 

>There isn't logic we could apply to it to only use item.title when a transform is detected 🤔
We might expand and add such logic to liquid. Or introduce new entire property for cart item, for instance `item.presentment_title`
Another option would be can redefine `item.title` to be the same as `item.product.title` instead of concatenation of `product title + variant title`
From representing point of view, what is the purpose of `cart.item.title`? would it make sense to be the same as product title for dawn or other themes?

Reply:
+1 to @ludoboludo's concerns: repeating the variant name in the product title is redundant for all buyers, including buyers using voice over. I am sure there are more clever outcome we can explore 🙏 

For example, when selling plans are available, it would be listed as a line item under the variant choices details. 

Curious to learn where this lands!

Reply:
Thank you @melissaperreault and @ludoboludo! 
Example with selling plans makes much sense, we probably should look into some conditional rendering here as well.

contextually for existing representation it makes sense to keep `item.product.title` and with listed variant details under. For all merchants that not using cart transform it is not needed regression. Overriding `product.title` doesn't seem right either, as we are not overriding product's title, but cart item's title. 

I will need to speak with product about this one, but suspect the best solution would be probably to introduce a new field here to have something like `item.title_override?` and `item.title_override` for this specific use case.

cc @dpwolf @ignacio-chiazzo

Reply:
@dpwolf I think we might need some UX advice here, 
would it make sense to have some kind of different behaviour in case if we have title override?

have something like this in default case (current behaviour)
```
- product title
   - variant title 1
   - variant title 2
```

and totally different rendering for case when we have title overrides, like skipping the variants values and have only

```
title override for line instead of product title - variant title 1/variant title 2
```

Should it even be on theme by default? or those changes have to be available for customization for each merchant for them to decide?